deploy/chart/templates/0000_50_olm_02-services.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
Shouldn't these have `ifs` so that they don't appear in upstream charts?

Reply:
fixed!